commons/utils/create_ssl.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+"""Script for generating custom certificate."""
+
+import os
+from OpenSSL import crypto
+
+# pylint: disable-msg=too-many-locals
+def generate_certificate(days=1, file_save_path=".", **kwargs):
+    """
+    This function is used to generate SSL certificate PEM file
+    :param days : number of days the certificate will expire
+    :param file_save_path:  The file path to be saved
+    :return:  final file path
+    """
+    email_address = kwargs.get("emailAddress", "administrator_test@seagate.com")
```

Comment:
what if some argument is missing ? will need to set it to some default

Reply:
if any argument is passed in generate_certificate call, it will take that. If not, it will take default value which is passed as second argument of kwargs.get()